src/components/Reports/GDPRPermissions/PermissionExpandableRow.test.js

```diff
@@ -40,6 +40,48 @@ describe('PermissionExpandableRow component', () => {
     },
   };
 
+  const dopplerApiClientDoubleWithOriginType = {
+    getSubscriberPermissionHistory: async () => {
+      return {
+        success: true,
+        value: {
+          items: [
+            {
+              subscriberEmail: 'test@fromdoppler.com',
+              fieldName: 'Permiso2',
+              fieldType: 'permission',
+              date: new Date('2021-02-10T15:22:00.000Z'),
+              value: 'true',
+              originIP: '181.167.226.47',
+              originType: 'Manual',
+            },
+          ],
+        },
+      };
+    },
+  };
+
+  const dopplerApiClientDoubleWithoutOriginType = {
+    getSubscriberPermissionHistory: async () => {
+      return {
+        success: true,
+        value: {
+          items: [
+            {
+              subscriberEmail: 'test@fromdoppler.com',
+              fieldName: 'Permiso2',
+              fieldType: 'permission',
+              date: new Date('2021-02-10T15:22:00.000Z'),
+              value: 'true',
+              originIP: '181.167.226.47',
+              originType: '',
```

Comment:
If we don't receive this field yet, maybe the real case to test this, it should be without this value at all.

Reply:
Sounds good to me, let's test the case when the property is not received. Thanks!